src/redux/reducers/addAccomReducer.js

```diff
@@ -1,10 +0,0 @@
-import { ADD_ACCOMMODATION_REQUEST } from "../types/accommodationTypes";
-
-export default function user(state = {}, action = {}) {
-  switch (action.type) {
-    case ADD_ACCOMMODATION_REQUEST:
-      return {};
-    default:
-      return state;
-  }
-}
```

Comment:
@hakoemmy you removed this file while rebasing yet it is used on a different feature, why?

Reply:
@AbdoulNuru The reason why I removed this reducer, I had the same reducer called ``` accommodationReducer``` that has everything that is related to accommodations. What I did, just brought your reducer into one. I thought this was doable since the reducers all use the same store key ```accommodation```.

Reply:
@hakoemmy okay if you did it knowingly and you tested that the other feature is still working as intended, that's fine.